PyQt5-stubs/QtWidgets.pyi

```diff
@@ -1038,7 +1038,7 @@ class QAction(QtCore.QObject):
 class QActionGroup(QtCore.QObject):
 
     class ExclusionPolicy(int): ...
-    # None = ... # type: 'QActionGroup.ExclusionPolicy'
+    None_ = ... # type: 'QActionGroup.ExclusionPolicy'
```

Comment:
I think this is an upstream bug.

```pycon
>>> QActionGroup.ExclusionPolicy.None_
```
```pytb
AttributeError: None_
```
but this works
```pycon
>>> getattr(QActionGroup.ExclusionPolicy, "None")
```
```
<ExclusionPolicy.None: 0>
```

Reply:
Same with `QOcspRevocationReason.None`

Reply:
Mmm, I guess leaving it commented makes more sense than lying.  As I recall there is `None_` elsewhere in upstream.  Thanks.  I still need to get my head into stubs and figure out how to test them.  I did see the other ticket about the pytest plugin and also saw another plugin.

Reply:
I'll reach out on the mailing list about these